mobile/lib/features/trade/position_list_item.dart

```diff
@@ -45,10 +48,18 @@ class _PositionListItemState extends State<PositionListItem> {
     formatter.maximumFractionDigits = 2;
 
     TradeTheme tradeTheme = Theme.of(context).extension<TradeTheme>()!;
+    PositionChangeNotifier positionChangeNotifier = context.watch<PositionChangeNotifier>();
+
+    // We're a bit conservative, we only enable action when we have both bid and ask
 
     // dart cannot promote...
     Position notNullPosition = widget.position!;
 
+    // XXX: Not sure whether ask or bid price here
+    bool hasPriceInOrderBook = notNullPosition.direction == Direction.long
```

Comment:
``` suggestion
   bool priceAvailable = notNullPosition.direction == Direction.long
```

I find the name `hasPriceInOrderBook` confusing - this has nothing to do with the orderbook, but with the app knowing about the current price.